tests/functional/rpcpb/etcd_config.go

```diff
@@ -58,8 +58,6 @@ var etcdFields = []string{
 	"LogOutputs",
 	"LogLevel",
 
-	"SocketReuseAddress",
```

Comment:
Out of curiosity -> Why  the options were problematic ?

Reply:
It's not supported in v3.4, so I removed it to have one code to test all v3.4, v3.5, v3.6 releases.